librz/il/_ct_opcodes.h

```diff
@@ -0,0 +1,327 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+/**
+ * \file _ct_opcodes.h
+ * \brief signatures of core theory opcodes
```

Comment:
Not sure that `opcode` is the correct term.

Reply:
@XVilka

Reply:
I think it's fine. It's the VM and VM tend to have opcodes. What is your suggestion? "operation"?